app/assets/javascripts/application.js

```diff
@@ -1,4 +1,5 @@
 //= require jquery/dist/jquery.min.js
 //= require click-tracker.js
+//= require take-whole-pot-calculator
```

Comment:
Fair enough, my thoughts were that it would be lower friction to get the code out there and then initialise it in the the next iteration. The require is now gone.